packages/pf3-component-mapper/src/form-fields/wizzard/button.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Button } from 'patternfly-react';
+
+const ButtonOverride = (props) => <Button style={{ marginLeft: 3 }} { ...props } />;
```

Comment:
Instead of doing this. Can you use the original button, give it default className with this styling? This spacing is required all over the PF3 form including the submit/cancel/reset buttons.